docs/user/enabling-firewall-logs.md

```
@@ -66,231 +67,77 @@ nft -a list chain inet fw4 input | grep fwlive-ping
-uci set firewall.@rule[10].log='1'
+# Adjust @rule[N] — uci show firewall | grep name=
+uci set firewall.@rule[N].log='1'
 uci commit firewall
 /etc/init.d/firewall reload
 ```
 
-In **`/etc/config/firewall`**:
+Or in `/etc/config/firewall`:
 
-```text
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)